GliaWidgets/View/Call/CallView.swift

```diff
@@ -56,10 +56,8 @@ class CallView: EngagementView {
 
     lazy var secondLabel: UILabel = {
         let label = UILabel()
-        #warning("""
-                 Need to take into account that this label is also used for visitor on hold,
-                 by introducing separate accessibility hints for each state (on-hold/call-duration).
-                 """)
+//	Note: Need to take into account that this label is also used for visitor on hold,
```

Comment:
Lets address this note so that reused label would have corresponding hint for on-hold and call-duration states.